mail/templates/course_run_unenrollment/body.html

```diff
@@ -0,0 +1,21 @@
+{% extends "email_base.html" %}
+
+{% block content %}
+<!-- 1 Column Text + Button : BEGIN -->
+  <tr>
+      <td style="background-color: #ffffff;">
+          <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+              <tr>
+                  <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
+                      <p style="margin: 0 0 10px;">Dear {{ user.name }},</p>
+                      <p style="margin: 0 0 10px;">
+                        You have been unenrolled in {{ enrollment.run }} starting {{ enrollment.run.start_date|date:"m D, Y"}}. The course will no longer appear on your <a href="{{ base_url }}{% url 'user-dashboard' %}">{{ site_name }} dashboard</a>.
```

Comment:
I'm not familiar with this date templating scheme, but I'd like something that looks like "January 2, 2019" -- i.e. month as a the full name followed by space, followed by day of month (no leading zero), followed by comma, followed by space, followed by four-digit year.

Reply:
@pdpinch updated the format. 
[Date-Reference](http://blog.tkbe.org/archive/date-filter-cheat-sheet/). 

#### Screens Updated:

<img width="999" alt="Screen Shot 2019-07-26 at 12 08 02 PM" src="https://user-images.githubusercontent.com/7334669/61933059-5a86fb80-af9e-11e9-9942-56173253311b.png">
<img width="998" alt="Jan 15" src="https://user-images.githubusercontent.com/7334669/61933061-5a86fb80-af9e-11e9-9a04-7e6d679c9b96.png">